### PR TITLE
Web doc improvment

### PR DESCRIPTION
**Pull request by atarkowska**
- adding wsgi-args https://github.com/openmicroscopy/openmicroscopy/pull/4352
- mod_wsgi required version

cc: @manics

### CODE REVIEW

File: omero/sysadmins/version-requirements.txt [hunk 1]
```diff
@@ -178,6 +178,12 @@ The planned changes for 5.2 depend upon our support of
     both of which will be dropped in 5.3 if CentOS 6 is dropped,
     otherwise they will remain deprecated.
 
+* mod_wsgi
+
+  * 3.5 recommended
+  * Rationale: Support for the version of mod_wsgi is down to the OS
+    vendor, not the mod_wsgi maintainers
```
Comment (mtbc) [inline omero/sysadmins/version-requirements.txt]: end with period

**Comment by will-moore** [comment]
As discussed, could mention security issues with mod_wsgi <3.5 and indicate that all versions since 3.5 are supported.

**Comment by atarkowska** [comment]
is that better?

File: omero/sysadmins/version-requirements.txt [hunk 2]
```diff
@@ -178,6 +178,14 @@ The planned changes for 5.2 depend upon our support of
     both of which will be dropped in 5.3 if CentOS 6 is dropped,
     otherwise they will remain deprecated.
 
+* mod_wsgi
+
+  * 3.5+ recommended
+  * Rationale: Choice of mod_wsgi version support is made by the OS vendors,
+    who may choose to continue with a older version with security patches
```
Comment (mtbc) [inline omero/sysadmins/version-requirements.txt]: "_an_ older version"

**Comment by hflynn** [comment]
@will-moore can you sign off on whether this is good to merge please?

**Comment by will-moore** [comment]
I think this is fine now. Good to merge.

**Comment by hflynn** [comment]
Actually waiting on a code Pr being merged apparently, will hold off for now

**Comment by atarkowska** [comment]
just added troubleshooting as stated in source https://github.com/iuscommunity-pkg/python27-mod_wsgi/blob/master/SOURCES/python27-mod_wsgi.conf (due to changes in https://github.com/openmicroscopy/ome-documentation/pull/1348#issuecomment-163579266)

**Comment by atarkowska** [comment]
@hflynn as decided in https://github.com/openmicroscopy/openmicroscopy/pull/4352#issuecomment-163583469 this PR is not relaying on anything now.

**Comment by hflynn** [comment]
@joshmoore confirms this is now as discussed, thanks.

**Pull request merged**